examples/demo/scikit_estimator.py

```diff
@@ -19,7 +19,7 @@
 
 # Model, strategy, oracle, and pipeline
 from pyrelational.models import ModelManager
-from pyrelational.oracle import DummyOracle
+from pyrelational.oracles import DummyOracle
 from pyrelational.pipeline.generic_pipeline import Pipeline
```

Comment:
I see what you mean by some of the conflicts with the new examples now, the DummyOracle has been superceeded with BenchmarkOracle and so on.